wai-aria/role/tab-roles.html

```diff
@@ -82,13 +82,9 @@
     <div role="tabpanel" data-testname="role is tabpanel as sibling to ul with child role none li elements" data-expectedrole="tabpanel" class="ex">Tab one's stuff</div>
     <div role="tabpanel" data-testname="role is tabpanel as sibling to ul with child role none li elements (duplicate)" data-expectedrole="tabpanel" class="ex">Tab two's stuff</div>
 
-<!--orphan tab semantics -->
-<button role="tab" data-testname="orphan button with tab role" data-expectedrole="button" class="ex">x</button>
-<span role="tab" data-testname="orphan span with tab role" class="ex-generic">x</span>
```

Comment:
Good catch that this one was not running due to the lack of `verifyGenericRoleByselector();`!